DEVELOPMENT.md

```diff
@@ -91,13 +91,18 @@ You can install the JDK from IntelliJ::
 
 If you use homebrew, you can do this instead:
 ```sh
-    # In getting set up initially, I made some mess, but I think this will work.
-    brew install temurin17
+    brew tap homebrew/cask-versions
+    brew install --cask temurin17
+
+    # If ~/.jenv/versions directory doesn't exist:
+    mkdir -p ~/.jenv/versions
     jenv add /Library/Java/JavaVirtualMachines/temurin-17.jdk/Contents/Home
  ```
 
 You may want to use jenv to set this JDK as your global setting to avoid surprises.
 
+To switch between jdk versions, modify `.bash_profile` and run `jenv global` as [described here](https://stackoverflow.com/a/43900696/6447189).
+
```

Comment:
Mention `jenv shell`. I have to use this currently to build the CLI in Java 11.

Reply:
To switch back and forth for CLI and WSM, I do `jenv global 17` and `jenv global 11`. What commands do you run?

Reply:
that works. I do `jenv shell openjdk64-11.0.9.1`, so it only applies to the current shell.

Reply:
Maybe point to jenv help instead of documenting here.

Reply:
Done. I also changed `jenv global` to generic `jenv`, so people can use whichever variant that want.